nemo_asr/run_canary_flash.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+export PYTHONPATH="..":$PYTHONPATH
+
+MODEL_IDs=("nvidia/canary-1b-flash")
```

Comment:
Kunal, no need for a new file just add to https://github.com/huggingface/open_asr_leaderboard/blob/0123743f72193698814998f175456a7eeccc10a2/nemo_asr/run_canary.sh#L5 as an additional model

Reply:
Done